Shared/Samples/Simplify geometry/SimplifyGeometryView.swift

```diff
@@ -0,0 +1,130 @@
+// Copyright 2025 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct SimplifyGeometryView: View {
+    /// A map with a light gray basemap style.
+    @State private var map = Map(basemapStyle: .arcGISLightGray)
+    /// A Boolean value indicating whether the geometry has been simplified.
+    @State private var isSimplified = false
+    /// The graphics overlay for the original polygon.
+    @State private var originalOverlay = GraphicsOverlay(graphics: [makePolygonGraphic()])
+    /// The graphics overlay for the simplified geometry.
+    @State private var resultOverlay = GraphicsOverlay()
+    /// The polygon graphic geometry.
+    private var graphicGeometry: Geometry? {
+        originalOverlay.graphics.first?.geometry
+    }
+    
+    var body: some View {
+        MapViewReader { mapView in
+            MapView(map: map, graphicsOverlays: [originalOverlay, resultOverlay])
+                .task {
+                    // Center the map on London.
+                    await mapView.setViewpointCenter(
+                        Point(
+                            x: -13150,
+                            y: 6710327,
+                            spatialReference: .webMercator
+                        ),
+                        scale: 25000
+                    )
+                }
+        }
+        .toolbar {
+            ToolbarItemGroup(placement: .bottomBar) {
+                Button("Simplify", role: .cancel) {
+                    simplifyGeometry()
+                }
+                .disabled(isSimplified)
+                
+                Button("Reset") {
+                    // Clear the simplified overlay and reset state.
+                    resultOverlay.removeAllGraphics()
+                    isSimplified = false
+                }
+                .disabled(!isSimplified)
+            }
+        }
+    }
+    
+    /// Checks to see whether the polygon geometry has been simplified, if not, simplifies the geometry and then displays it in filled red. 
+    private func simplifyGeometry() {
+        guard let original = originalOverlay.graphics.first?.geometry, !GeometryEngine.isSimple(original) else { return }
```

Comment:
The second statement in the `guard` statement should be indented.
```suggestion
        guard let original = originalOverlay.graphics.first?.geometry,
        !GeometryEngine.isSimple(original) else { return }
```